coops-ui/src/views/AnnualReport.vue

```diff
@@ -358,6 +441,8 @@ export default {
     } else {
       // else just load new page
       this.loadingMessage = `Preparing Your ${this.ARFilingYear} Annual Report`
+      // TODO: This filing code needs to be updated to ANNUAL_REPORT_BC when Payments codes have been updated.
```

Comment:
How are you tracking that this future work gets done? Ticket #?

Reply:
I will create a ticket right now!

Reply:
#2054

Placeholder ticket, until we can get the details from business regarding all the various codes 👍

Reply:
Perfect, thanks!